ci/cloudbuild/builds/cmake-oldest-deps.sh

```diff
@@ -36,6 +36,18 @@ io::log_h2 "Building"
 cmake --build cmake-out/build
 
 io::log_h2 "Testing"
-env -C cmake-out/build ctest -LE "integration-test" --parallel "$(nproc)"
+
+ctest_flags=(
+  # Print the full output on failures
+  --output-on-failure
+  # Skip tests with the "integration-test" label, the intent is to run only
+  # unit tests
+  -LE "integration-test"
```

Comment:
Perhaps this should be factored out so that the `ctest` invocation looks the same as in the other files?

Reply:
Done